docs/development-setup.md

```diff
@@ -192,3 +200,70 @@ container with credentials to push to the registry:
 ```
 $ osbs --config osbs.conf --instance local build -g https://github.com/TomasTomecek/hello-world-container -b master -u ${USER} -c hello-world
 ```
+
+## Configuring orchestration
+
+To test out orchestrated builds, you'll need another namespace to act as a
+test worker, and add the necessary permissions to it:
+
+```
+$ oc new-project worker01
+$ oc policy add-role-to-user edit system:serviceaccount:myproject:builder
+$ oc policy add-role-to-user edit system:serviceaccount:worker01:builder
+$ oc policy add-role-to-group system:build-strategy-custom system:authenticated
+$ oc project myproject
+```
+
+Then you'll need to create two configuration files. Create a file `reactor-conf/config.yaml`
+with the contents
+
+```
+version: 1
+clusters:
+  x86_64:
+  - name: worker01
+    max_concurrent_builds: 4
+    enabled: true
+```
+
+and a file `osbs-client-conf/osbs.conf` with the contents:
+
+```
+[general]
+verbose = true
+build_json_dir = <path to inputs in the build image>
+
+[worker01]
+openshift_url = https://<not-localhost-ip-address>:8443/
+builder_openshift_url = https://<not-localhost-ip-address>:8443/
+namespace = worker01
+use_kerberos = false
+verify_ssl = false
+use_auth = true
+registry_uri = <registry URL>
+```
+
+And create corresponding secrets:
+
+```
+$ oc secrets new osbs-client-conf osbs-client-con/osbs.conf
```

Comment:
At this point, it's probably safer to make these commands use: `oc -n myproject`
This will make these instructions more resilient as we may come back to it for reference and not execute command in exact order.